src/components/pages/AdminHome/AdminSidebar.js

```diff
@@ -1,31 +1,25 @@
 import { Menu } from 'antd';
 import React, { useState } from 'react';
-import '../../../styles/InstructorStyles/index.less';
+import { Link, useHistory } from 'react-router-dom';
+import '../../../styles/AdminStyles/index.less';
 import {
-  DesktopOutlined,
-  HomeOutlined,
-  ThunderboltOutlined,
-  PlusOutlined,
-  BellOutlined,
-  CalculatorOutlined,
-  FormOutlined,
-  DollarOutlined,
-  FolderOpenOutlined,
+  DashboardOutlined,
+  TableOutlined,
+  ProfileOutlined,
```

Comment:
Removed unneeded icons from imports and changed stylesheets to the semantically correct one.